start.py

```diff
@@ -912,6 +912,11 @@ def script_cli(start_arg=None):
     if start_arg is None:
         start_arg = StartArgScript.from_args(sys.argv)
 
+    # Remove first argument from sys.argv
+    # - start.py when running from code and ayon executable whne running
+    #   from build
```

Comment:
Fix typo `whne` -> `when` and restructure comment for readability?
```suggestion
    # - start.py when running from code 
    # - ayon executable when running from build
```